src/Switch-FTDI_FTD2XX/main.py

```diff
@@ -0,0 +1,143 @@
+# This Device Class is published under the terms of the MIT License.
+# Required Third Party Libraries, which are included in the Device Class
+# package for convenience purposes, may have a different license. You can
+# find those in the corresponding folders or contact the maintainer.
+#
+# MIT License
+#
+# Copyright (c) 2024 SweepMe! GmbH (sweep-me.net)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+# SweepMe! device class
+# Device: Arduino MCP4728
+
+import os
+import sys
+
+file_dir = os.path.dirname(__file__)
```

Comment:
The entire sys.path.append block can be replaced
by 

from pysweepme import FolderManager
FolderManager.addFolderToPATH()

which is a convenience function to add the path of the libs folders. However, the directory of __file__ will not be added because it is not needed.

However, the code will not work for SweepMe! 1.5.5 because in this version addFolderToPATH added also subfolders of libs to sys.path which does not work for the ftd2xx library because it contain a subpackage with the same name.

Therefore, I suggest to omit support for SweepMe! 1.5.5. by removing the "libs" folder that is only there for 1.5.5 compatibility.